sdk/formrecognizer/azure-ai-formrecognizer/migration-guide.md

```diff
@@ -0,0 +1,597 @@
+- Added methods for getting/listing operations of the past 24 hours, useful to track the status of model creation/copying operations and any resulting errors.
+
+Please refer to the [README][README] for more information on these new clients.
+
+## Important changes
+
+#### Instantiating clients
+
+In 3.x.x, the `FormRecognizerClient` and the `FormRecognizerAsyncClient` is instantiated via the `FormRecognizerClientBuilder`.
+
+In 4.x.x, the `FormRecognizerClient` and the `FormRecognizerAsyncClient`, has been replaced by the `DocumentAnalysisClient` and the `DocumentAnalysisAsyncClient` respectively and is instantiated via the [DocumentAnalysisClientBuilder][DocumentAnalysisClientBuilder].
+The sync and async operations are separated to [DocumentAnalysisClient][DocumentAnalysisClient] and [DocumentAnalysisAsyncClient][DocumentAnalysisAsyncClient].
+
+Instantiating FormRecognizerClient client with 3.x.x:
+```java
+FormRecognizerClient formRecognizerClient = new FormRecognizerClientBuilder()
+    .credential(new AzureKeyCredential("{key}"))
+    .endpoint("{endpoint}")
+    .buildClient();
+```
+
+Instantiating DocumentAnalysisClient client with 4.x.x:
+<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L52-L55 -->
+```java
+DocumentAnalysisClient documentAnalysisClient = new DocumentAnalysisClientBuilder()
+    .credential(new AzureKeyCredential("{key}"))
+    .endpoint("{endpoint}")
+    .buildClient();
+```
+Similarly, with 4.x.x, the `FormTrainingClient` and `FormTrainingAsyncClient` has been replaced by the `DocumentModelAdministrationClient` 
+and `DocumentModelAdministrationAsyncClient`, instantiated via the [DocumentModelAdministrationClientBuilder][DocumentModelAdministrationClientBuilder].
+The sync and async operations are separated to [DocumentModelAdministrationClient][DocumentModelAdministrationClient] and [DocumentModelAdministrationAsyncClient][DocumentModelAdministrationAsyncClient].
+
+Instantiating FormRecognizerClient client with 3.x.x:
+```java
+FormTrainingClient formTrainingClient = new FormTrainingClientBuilder()
+    .credential(new AzureKeyCredential("{key}"))
+    .endpoint("{endpoint}")
+    .buildClient();
+```
+
+Instantiating DocumentModelAdministrationClient client with 4.x.x:
+<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L62-L65 -->
+```java
+DocumentModelAdministrationClient documentModelAdminClient = new DocumentModelAdministrationClientBuilder()
+    .credential(new AzureKeyCredential("{key}"))
+    .endpoint("{endpoint}")
+    .buildClient();
+```
+
+#### Analyze documents
+Analyze data from certain types of common documents (such as receipts, invoices, business cards, or identity documents) 
+using prebuilt models or from custom documents using your own custom models and extract table and structured layout data.
```

Comment:
this sentence seems out of place. Maybe just say something about the differences between versions here?